Counties/Florida/Bay County/Scraper/tests/test_ScraperUtils.py

```diff
@@ -73,8 +73,8 @@ def test_parse_out_path_illegal_characters(self):
     def test_parse_out_path_valid(self):
         # Function should not affect valid length filenames and paths.
         normal_filename = 'document'
-        parsedPath = ScraperUtils.parse_out_path('C:\\Example\Path', normal_filename, 'pdf')
-        assert parsedPath == os.path.join('C:\\Example\Path', '{}.{}'.format(normal_filename, 'pdf'))
+        parsedPath = ScraperUtils.parse_out_path('C:\\Example\\Path', normal_filename, 'pdf')
+        assert parsedPath == os.path.join('C:\\Example\\Path', '{}.{}'.format(normal_filename, 'pdf'))
```

Comment:
This might conflict with my PR because I change this path into a raw string to address a depreciation warning about the same lines.